source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,122 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
+
+* The encryption algorithm (deterministic or random)
+* The data key
+* The BSON Type
+
+.. note::
+
+    A single data key can be used for all encrypted fields, or you can use individual data keys that are specific to each field.
+
+
+The MedcoMD engineers receive specific requirements for the fields of data. `Deterministic Encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#deterministic-encryption>`_ and `Random encryption <https://docs.mongodb.com/manual/core/security-client-side-encryption/#random-encryption>`_ are the two different methods of encrypting sensitive fields. The following table illustrates the data model of the Medco Management System.
+
+.. table:: 
+    :widths: 50 50
+
+    +-----------------------------+---------------------+----------------------------------------------+
+    |Field type                   |Encryption Algorithm | BSON Type                                    |
+    +=============================+=====================+==============================================+
+    | Name                        | Non-Encrypted       | String                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | SSN                         | Deterministic       |  Int                                         |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Blood Type                  | Random              |  String                                      |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Medical Records             | Random              |  Array                                       |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Policy Number    | Deterministic       |  Int (embedded inside `insurance` object)    |
+    +-----------------------------+---------------------+----------------------------------------------+
+    | Insurance: Provider         | Non-Encrypted       |  String (embedded inside `insurance` object) |
+    +-----------------------------+---------------------+----------------------------------------------+
+
+To automatically encrypt and decrypt create a JSON Schema using the information in the data model and pass the schema as a JSON object to the secure client to specify the encrypted field properties.
+In the JSON, you can specify `encryptMetadata <https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/#encryptmetadata-schema-keyword>`_ that is passed down to the fields under the ``properties``
```

Comment:
Thanks for revising this sentence. I think there are a few minor grammatical errors in this version that make this less clear than it could be:

1) "...you can specify encryptMetadata that..."

It is not clear that you are talking about JSON key-value pairs inside of the encryptMetadata object. If I said "In the office, you can specify desk", it's unclear whether I meant "select a desk to sit at" or "choose what materials it should be made of". 

2) Another point that makes it unclear what is being "passed down" is that you are using the singular helper verb "is" which implies the object `encryptMetadata` and not the properties within it.

Here's my sample sentence again:
```   
The encryptMetadata is used in this example to allow the encryption-specific properties to be inherited by fields defined in the properties section.
```

Reply:
How's this:

"To automatically encrypt and decrypt create a JSON Schema using the information in the data model and pass the schema as a JSON object to the secure client to specify the encrypted field properties.
In the JSON, you can specify `encryptMetadata <https://docs.mongodb.com/manual/reference/security-client-side-automatic-json-schema/#encryptmetadata-schema-keyword>`_ to specify top-level encryption Configuration options such as a keyId or encryption algorithm. Fields under the ``properties`` key inherit their encryption properties from encryptMetadata."

I think this clears up what is being inherited from encryptMetadata.

Reply:
what do you think @ccho-mongodb

Reply:
First sentence:
- Missing a comma: "To automatically encrypt and decrypt, create a JSON Schema..."
- This might be a run-on sentence, even with the comma.
- You also might consider mentioning the MongoDB driver since it executes the encryption/decryption based on the data in the JSON Schema.
- Since this paragraph is right before the code samples, maybe this should signal that the reader should be taking action.
- By putting together all of these comments, here's an example pair of sentences (to avoid the run-on): "Create a JSON Schema corresponding to the data model above. The MongoDB driver uses the schema to automatically manage the encryption and decryption of the specified fields."

Second sentence:
- I think `Configuration` should be lowercase.
- Parallel structure is needed. Corrected: "...such as a keyId or an encryption algorithm."

Third sentence:
- I think this could use more standardized language ("in objects" rather than "under ... key"). E.g. "Fields in the `properties` object inherit any encryption properties specified in the `encryptMetadata` object."